chromium_src/content/browser/browser_context.cc

```diff
@@ -62,6 +63,10 @@ namespace content {
 bool BrowserContext::IsTor() const {
   return false;
 }
+
+bool BrowserContext::IsAIChatAgent() const {
```

Comment:
do we actually need this on BrowserContext? I'm not sure if we need `IsTor` either and we should keep this scoped to Profile if possible.

Reply:
I guess the existing override is already for BrowserContext so I'll look at this in a follow-up